stable/magento/README.md

```diff
@@ -85,23 +98,69 @@ The above parameters map to the env variables defined in [bitnami/magento](http:
 >
 > The reserved IP address can be associated to the Magento service by specifying it as the value of the `magentoLoadBalancerIP` parameter while installing the chart.
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+### Installing on minikube
 
-```console
-$ helm install --name my-release \
-  --set magentoUsername=admin,magentoPassword=password,mariadb.mariadbRootPassword=secretpassword \
-    stable/magento
-```
+    $ helm inspect values stable/magento > values.yaml
```

Comment:
You should use command formatting syntax here, i.e.

`$ helm inspect values stable/magento > values.yaml`